spec/examples/pay_with_provider_advanced.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class PayWithProviderAdvanced < Actor
+  input :provider,
+        inclusion: {
```

Comment:
Since there is a `PayWithProviderInclusionAdvanced`, shouldn’t this be `in`?

```suggestion
        in: {
```

Reply:
Nope. In "advanced mode", there should be `inclusion.in`.